proto/substrait_extension/dist_plan.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+option go_package = "github.com/GreptimeTeam/greptime-proto/go/substrait_extension";
+
+package substrait_extension;
```

Comment:
Can you add a link to the substrait extension docs so that we understand how this works?

Reply:
I don't think we need to add the link here. The proto is the dependency of greptimedb, those internal structs or messages will not be used by other projects except greptimedb. The developer will find the detailed info there.

Reply:
I mean a link as a review comment or PR description.

For cross-refer to the substrait docs so that we know what we're doing here.

Reply:
Got it
Refer to 
https://substrait.io/

https://docs.greptime.com/contributor-guide/frontend/distributed-querying#dist-plan